torch/csrc/distributed/rpc/python_functions.cpp

```diff
@@ -32,7 +33,7 @@ py::object toPyObj(const Message& message) {
       Stack stack;
       stack.push_back(ret.value());
       {
-        pybind11::gil_scoped_acquire ag;
+        pybind11::gil_scoped_acquire acquire;
```

Comment:
"pybind11" has alias "py". 
https://github.com/pytorch/pytorch/blob/11a40410e755b1fe74efe9eaa635e7ba5712846b/torch/csrc/utils/pybind.h#L18

We can make it shorter. `py::gil_scoped_acquire acquire;`